aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -176,6 +185,15 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle) : AztecFormat
         joinStyleSpans(start, end)
     }
 
+    private fun applyMarkInlineStyle(start: Int = selectionStart, end: Int = selectionEnd, attrs: AztecAttributes = AztecAttributes()) {
+        val previousSpans = editableText.getSpans(start, end, MarkSpan::class.java)
+        previousSpans.forEach {
+            (it as MarkSpan).applyInlineStyleAttributes(editableText, start, end)
+
+            return@forEach
```

Comment:
We can safely omit the `return@forEach` here since it is a local return to the caller of the forEach loop that moves to next element of the array ([ref](https://kotlinlang.org/docs/returns.html#return-to-labels)).